app/models/claim_state_transition_reason.rb

```diff
@@ -2,60 +2,134 @@ class ClaimStateTransitionReason
   class ReasonNotFoundError < StandardError; end
   class StateNotFoundError < StandardError; end
 
-  attr_accessor :code, :description
+  attr_accessor :code, :description, :long_description
 
+  # rubocop:disable Metrics/LineLength
   TRANSITION_REASONS = {
     rejected: {
-      no_indictment: 'No indictment attached',
-      no_rep_order: 'No Magistrates’ representation order attached (granted before 1/8/2015)',
-      time_elapsed: 'Claim significantly out of time with no explanation.',
-      no_amend_rep_order: 'No amending representation order',
-      case_still_live: 'Case still live',
-      wrong_case_no: 'Incorrect case number',
-      wrong_maat_ref: 'Wrong MAAT reference',
-      other: 'Other'
+      no_indictment: {
```

Comment:
💭 Wouldn't this be better in a configuration file that this class reads from? (Feels like it, at least for me, a lot of it seems to be wording for a specific keyword, that as it stands, can't even be localized).

Reply:
yeah, i did consider taking this whole hash constant out as it is lookup data fundamentally. the options seem to me to be:

1. create a separate class/module just for the has
2. yamlize or other format it and load once on app start
3. store in db

first option the simplest - what do you think?

Reply:
💭 If you move the `short` and `long` to the locales, the list can remain here looking just like this, maybe:

 ```ruby
TRANSITION_REASONS = {
  rejected: %i[no_indictment no_rep_order time_elapsed no_amend_rep_order case_still_live wrong_case_no wrong_maat_ref other],
  disbursement: ...
}
```